autoload/previm.vim

```diff
@@ -208,7 +213,11 @@ function! s:do_external_parse(lines) abort
   endif
   let temp = tempname()
   call writefile(a:lines, temp)
-  return split(s:system(cmd . ' ' . s:escape_backslash(temp)), "\n")
+  if has('win32')
+    return split(s:system('python ' . exepath(cmd) . ' ' . s:escape_backslash(temp)), "\n")
+  else
+    return split(s:system(cmd . ' ' . s:escape_backslash(temp)), "\n")
+  endif
```

Comment:
Instead of changing these lines, how about changing
```vim
let candidates = ['rst2html.py', 'rst2html']
```
to
```vim
let candidates = ['python rst2html.py', 'python rst2html']
```

But I thought previm works also on windows.
@mattn is this change necessary?

Reply:
>is this change necessary?

No needed. It should work as tyru pointed.